examples/lines_bars_and_markers/linestyles.py

```diff
@@ -3,31 +3,44 @@
 Linestyles
 ==========
 
-This examples showcases different linestyles copying those of Tikz/PGF.
+This example showcases different linestyles copying those of Tikz/PGF.
```

Comment:
Can this be linked to example of Tikz/PGF linestlyes? Otherwise, there's no context.